verification/curator-service/ui/src/components/new-case-form-fields/PreexistingConditions.tsx

```diff
@@ -0,0 +1,32 @@
+import CaseFormValues from './CaseFormValues';
+import { FormikAutocomplete } from '../common-form-fields/FormikFields';
+import React from 'react';
+import Scroll from 'react-scroll';
+import { SelectField } from '../common-form-fields/FormikFields';
+import { useFormikContext } from 'formik';
+
+const hasPreexistingConditionsValues = [undefined, 'Yes', 'No'];
+export default function PreexistingConditions(): JSX.Element {
+    const { values, initialValues } = useFormikContext<CaseFormValues>();
+    return (
+        <Scroll.Element name="preexistingConditions">
+            <fieldset>
+                <legend>Pre-existing conditions</legend>
+                <SelectField
+                    name={`hasPreexistingConditions`}
```

Comment:
just "hasPreexistingConditions", no need for JS here

Reply:
Done